### DISCUSSION TITLE
Does VT try to sinchronize heating for different zones in over_climate?

### DISCUSSION THREAD

**Discussion by pieri70** [jmcollin78/versatile_thermostat#333] [Q&A]
Hello, may be stupid question..
I know that in over switch VT tries to shift the activations in order to minimize the number of switches active at a time.
Do you think it is possible (if not yet implemented) to make the contrary when using TRV (over climate)?
Consider this picture (not the left side, I had some problems with boiler and a TRV)

![image](https://github.com/jmcollin78/versatile_thermostat/assets/11612625/e329cfd4-854b-4c5f-8c4b-6ea5b13e4ccf)

At 5.30 my scheduler switches from eco to comfort so all VT move to heating.
After that time there seems to be no regular pattern and each VT go on heating/idle in different times, as a result the boiler  is almost always on.
Do you think it would be a good idea to delay/anticipate a little the change of status of different VT (different zones) so there is more synchronization and may be the boiler stay idle more time?

Ciao
Pietro

**Comment by jmcollin78** [comment]
The synchronisartion is limited to the switchs which are in the SAME VTherm. It is not possible to coordinate multiple VTherms. It have been requested many times but not possible as now.

If I understand well, your request is to synchronize VTherm (and the underlying in the same VTherm). So the answer in no, I cannot do that. I explain exactly why in another post but I don't find it.

**Comment by pieri70** [comment]
Ok thanks.
It's really good as is.
Ciao
